diff --git a/Makefile b/Makefile
index b336159f..7d1e071c 100644
--- a/Makefile
+++ b/Makefile
@@ -24,7 +24,7 @@ $(BUILDDIR)/bin/runc: $(RUNCBUILDDIR)
 	mkdir -p $(BUILDDIR)/bin
 	mv $(RUNCBUILDDIR)/runc $(BUILDDIR)/bin/runc
 
-internal/binutils/runc.go: $(BUILDDIR)/bin/runc
+internal/binutils/runc.go:
 	go-bindata -tags \!noembed -pkg binutils -prefix "$(BUILDDIR)/bin" -o $(CURDIR)/internal/binutils/runc.go $(BUILDDIR)/bin
 	gofmt -s -w $(CURDIR)/internal/binutils/runc.go
 
