diff --git a/setfiles/Makefile b/setfiles/Makefile
index 84ffb08b..31bc07db 100644
--- a/setfiles/Makefile
+++ b/setfiles/Makefile
@@ -5,8 +5,11 @@ SBINDIR ?= /sbin
 MANDIR = $(PREFIX)/share/man
 AUDITH ?= $(shell test -f /usr/include/libaudit.h && echo y)
 
-CFLAGS ?= -g -Werror -Wall -W
-override LDLIBS += -lselinux -lsepol -lpthread
+# override with -lfts when building on Musl libc to use fts-standalone
+FTS_LDLIBS ?=
+
+CFLAGS ?= -g -Werror -Wall -W $(FTS_LDLIBS)
+override LDLIBS += -lselinux -lsepol -lpthread $(FTS_LDLIBS)
 
 ifeq ($(AUDITH), y)
 	override CFLAGS += -DUSE_AUDIT
