diff --git a/Makefile b/Makefile
index ebcc9d7c..b57ad620 100644
--- a/Makefile
+++ b/Makefile
@@ -33,7 +33,7 @@ CFLAGS ?= -Wall -Wshadow -Wextra -Wundef -Wmissing-format-attribute -Wcast-align -Wstrict-prototypes -Wpointer-arith -Wunused
 override CFLAGS += -D_GNU_SOURCE -D_FILE_OFFSET_BITS=64
 override LDLIBS += -lsepol

-all: $(SECILC) $(SECIL2CONF) $(SECIL2TREE) $(SECILCHECK) man
+all: $(SECILC) $(SECIL2CONF) $(SECIL2TREE) $(SECILCHECK)

 $(SECILC): $(SECILC_OBJS)
 	$(CC) $(CPPFLAGS) $(CFLAGS) $(LDFLAGS) -o $@ $^ $(LDLIBS)
@@ -67,17 +67,12 @@ $(SECILCHECK_MANPAGE): $(SECILCHECK_MANPAGE).xml
 $(SECILCHECK_MANPAGE): $(SECILCHECK_MANPAGE).xml
 	$(XMLTO) man $(SECILCHECK_MANPAGE).xml

-install: all man
+install: all
 	-mkdir -p $(DESTDIR)$(BINDIR)
-	-mkdir -p $(DESTDIR)$(MANDIR)/man8
 	install -m 755 $(SECILC) $(DESTDIR)$(BINDIR)
 	install -m 755 $(SECIL2CONF) $(DESTDIR)$(BINDIR)
 	install -m 755 $(SECIL2TREE) $(DESTDIR)$(BINDIR)
 	install -m 755 $(SECILCHECK) $(DESTDIR)$(BINDIR)
-	install -m 644 $(SECILC_MANPAGE) $(DESTDIR)$(MANDIR)/man8
-	install -m 644 $(SECIL2CONF_MANPAGE) $(DESTDIR)$(MANDIR)/man8
-	install -m 644 $(SECIL2TREE_MANPAGE) $(DESTDIR)$(MANDIR)/man8
-	install -m 644 $(SECILCHECK_MANPAGE) $(DESTDIR)$(MANDIR)/man8

 doc:
 	$(MAKE) -C docs
