diff --git a/meson.build b/meson.build
index afda4c0431ae..661663f12f5e 100644
--- a/meson.build
+++ b/meson.build
@@ -1060,7 +1060,11 @@ threads = dependency('threads')
 librt = cc.find_library('rt')
 libm = cc.find_library('m')
 libdl = cc.find_library('dl')
-libcrypt = dependency('libcrypt')
+libcrypt = dependency('libcrypt', 'libxcrypt', required : false)
+if not libcrypt.found()
+        # fallback to use find_library() if libcrypt is provided by glibc, e.g. for LibreELEC.
+        libcrypt = cc.find_library('crypt')
+endif
 libcap = dependency('libcap')

 # On some architectures, libatomic is required. But on some installations,
